index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@700&family=Sofia+Sans:wght@200&display=swap');
+        @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Sofia+Sans:wght@200&display=swap');
+        @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@700&family=Sofia+Sans:wght@200&display=swap');
+        @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto:wght@700&family=Sofia+Sans:wght@200&display=swap');
+
+        .crnr-img {
+            position: relative;
+            left: 12%;
+            top: -60px;
+            width: 150px;
+
+        }
+        .buttom h1{
+    margin: 0 !important;
+}
+.buttom ul{
+    margin: 0 !important;
+}
+    </style>
```

Comment:
As a best practice, kindly move inline styles to an external stylesheet to allow for reusability.